src/common/docker-compose/template.ts

```diff
@@ -7,7 +7,6 @@ interface XBakeConfig {
 
 export interface DockerServiceBuild {
   context?: string;
-  buildpack?: boolean;
```

Comment:
Reason I removed this is because its not part of the docker-compose spec. I wanted what we generate to be valid.